src/gen/src/IBDgen.cc

```diff
@@ -29,9 +29,12 @@ void IBDgen::UpdateFromDatabaseIndex()
   Emax = libd->GetD("emax");
   // Flux function
   rmpflux.Set(libd->GetDArray("spec_e"), libd->GetDArray("spec_flux"));
+  std::vector<double> flux = libd->GetDArray("spec_flux");
+  FluxMax = 1.1 * (*max_element(flux.begin(), flux.end()));
```

Comment:
Why is there a factor of 1.1 here?

Reply:
That's my lazy way of adding epsilon to 1. The value is the maximum y with which to test in the MC integration, since random draw [0, 1) then max y should really be (1+ε) so that it's possible to get the max. It could be 1.0000001 and still be fine, but 1.1 seemed reasonable too. Technically it does make the random draw less efficient since there is more failure space than before, though that isn't a bottle neck. It should be noted that since we use a binned flux, calculating and using the CDF in this case would get us a factor of 2 or 3 more performant random draw.